cloudbuild-k8s-generator.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env python
+#
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+CLOUDBUILD_CONFIG = 'cloudbuild.yaml'
+
+HEADER = """timeout: 1200s # 20m
```

Comment:
Could we stamp a banner at the top to mark this file as autogenerated?

Something like:

```
//////////////////////////////////////////////////////////////////////////////////
// This file is generated by cloudbuild-k8s-generator.py. Do not manually edit. //
//////////////////////////////////////////////////////////////////////////////////
```

Reply:
Genius!